### ISSUE TITLE
Implementation Report

### ISSUE THREAD

**Issue by AlexDawsonUK** [w3c/sustyweb#112]
**Issue:** Interoperability is something we’ve been pulled up on. We need to show our stuff can be implemented within sites.

**Solution:** We could create an implementation report ([see WCAG's](https://www.w3.org/WAI/WCAG21/implementation-report/)) as our work gets wider adoption to showcase maturity (EcoGrader). Also worth noting that while tooling will allow us to show this on a wide scale (due to mass testing using a static methodology), there is nothing to prevent us using early adopters and those applying the WSGs to create a report based upon those works.

**Note:** WCAG has a [reporting tool](https://www.w3.org/WAI/eval/report-tool/) that could be something to add to the bucket list for STAR.

**Comment by AlexDawsonUK** [comment]
**Additional:** Creating some kind of tooling to take the measurability index mentioned in #104 and testing those metrics against a large subset of websites to produce an implementation report would help showcase interoperability (without bias being introduced). This could be produced by our group or a third-party but should be open source and actively developed if possible to ensure compliance with specification changes as they develop.

**Comment by AlexDawsonUK** [comment]
This previously open issue is now [under the review of the Interest Group](https://github.com/w3c/sustainableweb-wsg/issues/9), discussion and resolution will continue there.
